cloudsql/postgres/database-sql/cmd/app/.gcloudignore

```diff
@@ -0,0 +1,25 @@
+# This file specifies files that are *not* uploaded to Google Cloud
```

Comment:
Do we need a .gcloudignore here? Doesn't it use .gitignore by default?

Reply:
This is a mistaken addition. I'll remove it when I send the postgres PR.